lib/cog/chat/slack/connector.ex

```diff
@@ -300,12 +300,21 @@ defmodule Cog.Chat.Slack.Connector do
   defp classify_id(id, slack),
     do: id |> String.replace(~r/(^<|>$)/, "") |> do_classify_id(slack)
 
-  # TODO: use Slack.Lookups to verify the "C" and "U" cases here, to
```

Comment:
Yeah, `Slack.Lookups` are super naive. They don't take into account rooms or users not existing, they just crash. I'm basically doing what the functions in `Slack.Lookups` do, I'm just verifying the data. Alternatively I could just rescue, but that seemed a bit dirty.